tests/test_allo.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+from allo.allo import Dojo
+
+
+class TestCreateRoom(unittest.TestCase):
+    def test_create_room_successfully(self):
```

Comment:
There is a method called setUp that can set up your classes for you so you do not have to create new classes for each different test